lib/mix/tasks/compile/live_view.ex

```diff
@@ -0,0 +1,99 @@
+defmodule Mix.Tasks.Compile.LiveView do
+  use Mix.Task
+  @recursive true
+
+  alias Mix.Task.Compiler.Diagnostic
+
+  @doc false
+  def run(args) do
+    {compile_opts, _argv, _errors} = OptionParser.parse(args, strict: [return_errors: :boolean])
```

Comment:
We don't want strict here. The options given to `mix compile` is shared across all compilers. :)

```suggestion
    {compile_opts, _, _} = OptionParser.parse(args, switches: [return_errors: :boolean])
```